Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Utilities/PlayerData.cs

```diff
@@ -106,4 +101,39 @@ public static void Clear()
     /// Saved Wallet Connect Config used for restoring session (Remember Me) Implementation.
     /// </summary>
     [JsonProperty] public WalletConnectConfig WalletConnectConfig { get; set; }
+
+#if UNITY_EDITOR
+    [MenuItem("Tools/Player Data/Remove")]
+    public static void RemovePlayerData()
+    {
+        if (File.Exists(Instance.Path))
+        {
+            File.Delete(Instance.Path);
+        }
+        else
+        {
+            Debug.LogError($"{nameof(PlayerData)} not found at {Instance.Path}.");
+        }
+        
+        Debug.Log($"{nameof(PlayerData)} Removed.");
+    }
+    
+    [MenuItem("Tools/Player Data/Clear")]
+    public static void ClearPlayerData()
+    {
+        Clear();
+    }
+    
+    [MenuItem("Tools/Open Persistent Data Path")]
+    public static void OpenPersistentDataPath()
+    {
+        ProcessStartInfo startInfo = new ProcessStartInfo
+        {
+            Arguments = Application.persistentDataPath,
+            FileName = "explorer.exe",
```

Comment:
This will not work on MacOS.

Reply:
```string path = Application.persistentDataPath;

        
        if (Application.platform == RuntimePlatform.WindowsPlayer || Application.platform == RuntimePlatform.WindowsEditor)
        {
            Process.Start("explorer.exe", path);
        }
        else if (Application.platform == RuntimePlatform.LinuxPlayer || Application.platform == RuntimePlatform.LinuxEditor)
        {
            Process.Start("xdg-open", path);
        }
        else if (Application.platform == RuntimePlatform.OSXPlayer || Application.platform == RuntimePlatform.OSXEditor)
        {
            Process.Start("open", path);
        }

Reply:
that's a good point! done.

Reply:
only change I made to your snippet above is removed payer cases since this is an editor only function and removed `explorer.exe` since it kept opening documents instead on my end.